extensions/unknown.yaml

```diff
@@ -0,0 +1,71 @@
+%YAML 1.2
+---
+types:
+  - name: unknown
+scalar_functions:
+  -
```

Comment:
Why the extra newline?

Reply:
Removed, was a style choice that I don't care much about. The other extensions use this to make it easier to read.